### PR TITLE
add adjustdirection option

### PR DESCRIPTION
**Pull request by hrhotz**

### CODE REVIEW

File: tools/mafft/mafft.xml [hunk 1]
```diff
@@ -64,6 +68,10 @@
             <when value="mafft-qinsi"/>
             <when value="mafft-xinsi"/>
         </conditional>
+        <param name="adjustdirection" type="select" display="radio" label="Direction of nucleotide sequences" help="generate reverse complement sequences, as necessary, and align them together with the remaining sequences">
+            <option value="adjustdirection">adjust direction</option>
```
Comment (bgruening) [inline tools/mafft/mafft.xml]: @hrhotz any reason to not use a boolean checkbox?

Comment (hrhotz) [inline tools/mafft/mafft.xml]: yes, there is a third option (--adjustdirectionaccurately) which we might add at a later stage

Comment (bgruening) [inline tools/mafft/mafft.xml]: Ok, nice!
Than my last concern. If you use `--adjustdirection` in the option tag inside of your select you can simply use `$adjustdirection` in the command section.

File: tools/mafft/mafft.xml [hunk 2]
```diff
@@ -64,6 +68,10 @@
             <when value="mafft-qinsi"/>
             <when value="mafft-xinsi"/>
         </conditional>
+        <param name="adjustdirection" type="select" display="radio" label="Direction of nucleotide sequences" help="generate reverse complement sequences, as necessary, and align them together with the remaining sequences">
```
Comment (nsoranzo) [inline tools/mafft/mafft.xml]: s/generate/Generate/

Comment (hrhotz) [inline tools/mafft/mafft.xml]: fixed - thank you for spotting it

File: tools/mafft/mafft.xml (hunk 1 above)
Comment (hrhotz) [inline tools/mafft/mafft.xml]: sure - I have changed it.

**Comment by bgruening** [comment]
Sehr schön! Danke Hans-Rudolf!

**Comment by bgruening** [comment]
TS is updated!

**Pull request merged**